packages/editor/src/components/post-publish-panel/test/__snapshots__/index.js.snap

```diff
@@ -1,197 +1,733 @@
+.emotion-0 *::after {
+  box-sizing: inherit;
+}
+
+.emotion-2 {
+  margin-bottom: calc(4px * 2);
+}
+
+.components-panel__row .emotion-2 {
+  margin-bottom: inherit;
+}
+
+.emotion-4 {
+  font-size: 11px;
+  font-weight: 500;
+  line-height: 1.4;
+  text-transform: uppercase;
+  display: inline-block;
+  margin-bottom: calc(4px * 2);
+  padding: 0;
+}
+
+<div>
   <div
-    className="editor-post-publish-panel__header"
+    class="editor-post-publish-panel"
   >
-    <ForwardRef(Button)
-      icon={
-        <SVG
+    <div
+      class="editor-post-publish-panel__header"
+    >
+      <button
+        aria-label="Close panel"
+        class="components-button has-icon"
+        type="button"
+      >
+        <svg
+          aria-hidden="true"
+          focusable="false"
+          height="24"
           viewBox="0 0 24 24"
+          width="24"
           xmlns="http://www.w3.org/2000/svg"
         >
-          <Path
+          <path
             d="M12 13.06l3.712 3.713 1.061-1.06L13.061 12l3.712-3.712-1.06-1.06L12 10.938 8.288 7.227l-1.061 1.06L10.939 12l-3.712 3.712 1.06 1.061L12 13.061z"
           />
-        </SVG>
-      }
-      label="Close panel"
-    />
-  </div>
-  <div
-    className="editor-post-publish-panel__content"
-  >
-    <WithSelect(PostPublishPanelPostpublish)
-      focusOnMount={true}
-    />
-  </div>
-  <div
-    className="editor-post-publish-panel__footer"
-  >
-    <CheckboxControl
-      label="Always show pre-publish checks."
-    />
+        </svg>
+      </button>
+    </div>
+    <div
+      class="editor-post-publish-panel__content"
+    >
+      <div
+        class="post-publish-panel__postpublish"
+      >
+        <div
+          class="components-panel__body post-publish-panel__postpublish-header is-opened"
+        >
+          <a>
+            (no title)
+          </a>
+           
+          is now live.
+        </div>
+        <div
+          class="components-panel__body is-opened"
+        >
+          <p
+            class="post-publish-panel__postpublish-subheader"
+          >
+            <strong>
+              What’s next?
+            </strong>
+          </p>
+          <div
+            class="post-publish-panel__postpublish-post-address-container"
+          >
+            <div
+              class="components-base-control post-publish-panel__postpublish-post-address emotion-0 emotion-1"
+            >
+              <div
+                class="components-base-control__field emotion-2 emotion-3"
+              >
+                <label
+                  class="components-base-control__label emotion-4 emotion-5"
+                  for="inspector-text-control-0"
+                >
+                  undefined address
+                </label>
+                <input
+                  class="components-text-control__input"
+                  id="inspector-text-control-0"
+                  readonly=""
+                  type="text"
+                  value="undefined"
```

Comment:
Another `undefined` 🤔  I'm not sure this is valid HTML at all ?

Reply:
Addressed as part of https://github.com/WordPress/gutenberg/commit/9b8da201a0c63fb1fc7be29031bf0215277b8731.